packages/react-native-reanimated/apple/reanimated/apple/native/PlatformDepMethodsHolderImpl.mm

```diff
@@ -2,11 +2,21 @@
 #import <reanimated/apple/READisplayLink.h>
 #import <reanimated/apple/REANodesManager.h>
 #import <reanimated/apple/REASlowAnimations.h>
+#import <reanimated/apple/REAUIView.h>
 #import <reanimated/apple/RNGestureHandlerStateManager.h>
 #import <reanimated/apple/keyboardObserver/REAKeyboardEventObserver.h>
 #import <reanimated/apple/native/SetGestureState.h>
 #import <reanimated/apple/sensor/ReanimatedSensorContainer.h>
 
+#import <React/RCTComponentViewProtocol.h>
+#import <React/RCTComponentViewRegistry.h>
+#import <React/RCTMountingManager.h>
+
+#if __has_include(<rnscreens/RNSScreen.h>)
+#define HAS_SCREENS
```

Comment:
Does it work with `use_frameworks`? Will this work if we decide to prebuild RNReanimated and RNScreens?